### PR TITLE
i18n translation

### PR DESCRIPTION
**Pull request by davidDoskalovich07**

### CODE REVIEW

File: apps/kol-amelamdim/public/locales/en/common.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "optimized": {
+      "question": "Do you like to unleash some super powers to have all side optimized translations?",
+      "answer": "Then you may have a look at <1>this blog post</1>.",
+      "link": "https://locize.com/blog/next-i18next/"
+    },
+    "ssg": {
+      "question": "Do you want to use SSG (next export)?",
+      "answer": "Then you may have a look at <1>this blog post</1>.",
+      "link": "https://locize.com/blog/next-i18n-static/"
+    }
```
Comment (N4tty93) [inline apps/kol-amelamdim/public/locales/en/common.json]: I think we can remove it

**Review by N4tty93 — APPROVED** [review]

File: libs/types/src/index.ts [hunk 2]
```diff
@@ -6,62 +6,47 @@ export enum Category {
...
-    enName: 'Mivhanim',
-    hebName: 'מבחנים',
+    en: 'Mivhanim',
+    he: 'מבחנים',
   },
   {
     URL: 'art_and_activities',
-    enName: 'Art andactivities',
-    hebName: 'דפי יצירה ופעילות',
+    en: 'Art andactivities',
```
Comment (N4tty93) [inline libs/types/src/index.ts]: space after "and"

**Review by N4tty93 — APPROVED** [review]

File: apps/kol-amelamdim/public/locales/en/common.json (hunk 1 above)
Comment (davidDoskalovich07) [reply apps/kol-amelamdim/public/locales/en/common.json]: Resolved.

File: libs/types/src/index.ts (hunk 2 above)
Comment (davidDoskalovich07) [reply libs/types/src/index.ts]: Resolved.

**Pull request merged**